core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -565,13 +568,24 @@ public synchronized void checkpoint() throws IOException {
    * The caller is responsible for detecting and responding to leadership changes.
    */
   private void catchUp(JournalStateMachine stateMachine, LocalFirstRaftClient client)
-      throws TimeoutException, InterruptedException {
+          throws TimeoutException, InterruptedException {
     long startTime = System.currentTimeMillis();
     long waitBeforeRetry = ServerConfiguration.global()
         .getMs(PropertyKey.MASTER_EMBEDDED_JOURNAL_CATCHUP_RETRY_WAIT);
     // Wait for any outstanding snapshot to complete.
     CommonUtils.waitFor("snapshotting to finish", () -> !stateMachine.isSnapshotting(),
         WaitForOptions.defaults().setTimeoutMs(10 * Constants.MINUTE_MS));
+    OptionalLong endCommitIndex = OptionalLong.empty();
+    try {
+      endCommitIndex = getGroupInfo().getCommitInfos().stream()
+          .mapToLong(RaftProtos.CommitInfoProto::getCommitIndex).min();
+    } catch (IOException e) {
+      LOG.warn("Failed to get raft log information before replay."
+          + " Replay statistics will not be available:", e);
```

Comment:
I think we want to avoid dumping a full stack trace on warn, this can be pretty concerning for admins when it is a recoverable issue.

Reply:
Good point. I'll change this to `LogUtils.warn` which should just grab the exception message instead, unless debug logs are enabled

Reply:
fixed